modules/core/src/main/scala/org/tessellation/l0/SnapshotStorage.scala

```diff
@@ -0,0 +1,25 @@
+package org.tessellation.l0
+
+import cats.effect.kernel.{Async, Ref}
+import cats.syntax.functor._
+
+import org.tessellation.schema.height.Height
+
+trait SnapshotStorage[F[_]] {
+  def storeSnapshot(snapshot: DAGSnapshot): F[Unit]
+  def getLastSnapshotHeight: F[Height]
+}
+
+object SnapshotStorage {
+
+  def make[F[_]: Async]: SnapshotStorage[F] =
+    new SnapshotStorage[F] {
+      private val lastSnapshot: Ref[F, Option[DAGSnapshot]] = Ref.unsafe(None)
+
+      override def storeSnapshot(snapshot: DAGSnapshot): F[Unit] = lastSnapshot.set(Some(snapshot))
```

Comment:
Should storage on disk be separated from storage in memory?

Reply:
Can we store it only on disk? 🤔 
Maybe we can use a database here, instead of file storage

Reply:
Do you mean in memory DB or persisted?

Reply:
persisted 🤔

Reply:
Ok DB works for me. Much better than storing a file. Also, I am thinking if we still need some `Ref` or in-memory information about the last snapshot if we decide to store *whole* snapshot in DB. Probably not.